src/main/java/frc/robot/turret/ITurret.java

```diff
@@ -5,6 +5,8 @@
 public interface ITurret {
```

Comment:
i like when there is one line space from start {

of class and before end

}, but if you don't like it its ok but keep it the same in all of your classes